lib/internal/per_context/domexception.js

```diff
@@ -49,12 +49,25 @@ const disusedNamesSet = new SafeSet()
   .add('ValidationError');
 
 class DOMException {
-  constructor(message = '', name = 'Error') {
+  constructor(message = '', options = 'Error') {
     ErrorCaptureStackTrace(this);
-    internalsMap.set(this, {
-      message: `${message}`,
-      name: `${name}`
-    });
+
+    if (options && typeof options === 'object') {
+      const { name } = options;
```

Comment:
```suggestion
      const { name, cause } = options;
```

Reply:
@flakey5 ... either destruct `cause` out here or change the `ObjectDefineProperty` below to `value: options.cause`